hclsyntax/fuzz/README.md

```diff
@@ -1,85 +1,62 @@
 # hclsyntax fuzzing utilities
 
-This directory contains helper functions and corpuses that can be used to
-fuzz-test the `hclsyntax` parsers using [go-fuzz](https://github.com/dvyukov/go-fuzz).
+This directory contains helper functions and corpora that can be used to
+fuzz-test the `hclsyntax` parsers using Go's native fuzz testing capabilities.
 
-## Work directory
+Please see https://go.dev/doc/fuzz/ for more information on fuzzing.
 
-`go-fuzz` needs a working directory where it can keep state as it works.  This
-should ideally be in a ramdisk for efficiency, and should probably _not_ be on
-an SSD to avoid thrashing it. Here's how to create a ramdisk:
+## Prerequisites
+* Go 1.18
 
-### macOS
-
-```
-$ SIZE_IN_MB=1024
-$ DEVICE=`hdiutil attach -nobrowse -nomount ram://$(($SIZE_IN_MB*2048))`
-$ diskutil erasevolume HFS+ RamDisk $DEVICE
-$ export RAMDISK=/Volumes/RamDisk
-```
+## Running the fuzzer
 
-### Linux
+Each exported function in the `hclsyntax` package has a corresponding fuzz test.
+These can be run one at a time via `go test`:
 
 ```
-$ mkdir /mnt/ramdisk
-$ mount -t tmpfs -o size=1024M tmpfs /mnt/ramdisk
-$ export RAMDISK=/mnt/ramdisk
+$ cd fuzz
+$ go test -fuzz -run=FuzzParseTemplate
+$ go test -fuzz -run=FuzzParseTraversalAbs
+$ go test -fuzz -run=FuzzParseExpression
+$ go test -fuzz -run=FuzzParseConfig
```

Comment:
These commands didn't work for me with the latest gotip build. What worked was:

```shellsession
$ go test -fuzz FuzzParseConfig
```

Reply:
Whoops, thanks. I don't think the version I wrote ever worked...